pallets/creditcoin/src/mock.rs

```diff
@@ -292,3 +295,16 @@ pub fn pending_rpc_request(
 		..Default::default()
 	}
 }
+
+#[test]
+fn offchain_worker_should_log_when_authority_is_missing() {
+	ExtBuilder::default().build_offchain_and_execute(|| {
+		System::set_block_number(1);
+
+		begin_capture();
+
+		Creditcoin::offchain_worker(System::block_number());
+
+		assert_eq!(pop_captured().unwrap().message(), "No authority, skipping off chain work");
```

Comment:
@nathanwhit how does logging in substrate get initialized ? I don't see `set_logger` being called anywhere and I get an error here:
```
called `Result::unwrap()` on an `Err` value: SetLoggerError(())
```